coral-trino/src/main/java/com/linkedin/coral/trino/trino2rel/TrinoToRelConverter.java

```diff
@@ -44,8 +44,8 @@
 public class TrinoToRelConverter extends ToRelConverter {
   private final ParseTreeBuilder parseTreeBuilder = new ParseTreeBuilder();
   private final ParserVisitorContext parserVisitorContext = new ParserVisitorContext();
-  private final Multimap<String, Function> functionRegistry = new StaticHiveFunctionRegistry().getRegistry();
-
+  private final HiveFunctionResolver functionResolver =
```

Comment:
This is will become just `FunctionResolver` in https://github.com/linkedin/coral/issues/235